client/my-sites/stats/stats-insights/index.jsx

```diff
@@ -72,51 +81,72 @@ export default React.createClass( {
 							beforeNavigate={ this.updateScrollPosition } />;
 		}
 
-		return (
-			<div className="main main-column" role="main">
-				<SidebarNavigation />
-				<StatsNavigation section="insights" site={ site } />
-				<div id="my-stats-content">
-					<PostingActivity />
-					<LatestPostSummary site={ site } />
-					<TodaysStats
-						siteId={ site ? site.ID : 0 }
-						period="day"
-						date={ summaryDate }
-						path={ '/stats/day' }
-						title={ this.translate( 'Today\'s Stats' ) }
-					/>
-					<AllTime allTimeList={ allTimeList } />
-					<MostPopular insightsList={ insightsList } />
-					<DomainTip siteId={ site ? site.ID : 0 } event="stats_insights_domain" />
-					<div className="stats-nonperiodic has-recent">
-						<div className="module-list">
-							<div className="module-column">
-								<Comments
-									path={ 'comments' }
-									site={ site }
-									commentsList={ commentsList }
-									followList={ followList }
-									commentFollowersList={ commentFollowersList } />
-								{ tagsList }
-							</div>
-							<div className="module-column">
-								<Followers
-									path={ 'followers' }
-									site={ site }
-									wpcomFollowersList={ wpcomFollowersList }
-									emailFollowersList={ emailFollowersList }
-									followList={ followList } />
-								<StatsModule
-									path={ 'publicize' }
-									moduleStrings={ moduleStrings.publicize }
-									site={ site }
-									dataList={ publicizeList } />
-							</div>
+		const content = (
+			<div id="my-stats-content">
```

Comment:
I see. I wasn't aware that the stats section wasn't using the preferred components already. So, just to make sure I'm clear, this suggestion is irregardless of this particular PR (in other words, it would be recommended even if they scrapped the whole "first view" stuff).